baseplate/clients/thrift.py

```diff
@@ -168,11 +168,16 @@ def _call_thrift_method(self: Any, *args: Any, **kwargs: Any) -> Any:
         last_error = None
 
         for time_remaining in self.retry_policy:
-            span = self.server_span.make_child(trace_name)
-            span.start()
-
             try:
                 with self.pool.connection() as prot:
+                    span = self.server_span.make_child(trace_name)
+                    span.set_tag("slug", self.namespace)
```

Comment:
The spec specifies a slug, and I chose the namespace string to be that slug - but I'm not actually completely sure that it's the right id to use. It shows up in the tests as "example-service" which looks good, but it might be used differently other places?

Reply:
Good enough for me. Thanks for explaining